public/assets/css/ci-theme.css

```
@@ -239,6 +239,16 @@ a {
 
 /* HEADER =================================================================== */
 
+header-for-home {

```

Shouldn't this be a class or an ID? Like:
```suggestion
.header-for-home {
```